infrastructure/emergency-backup-metadata.sh

```diff
@@ -113,7 +113,15 @@ else
   ROOT_PATH=$(cd "$ROOT_PATH" && pwd)
 fi
 
+# Find and remove all empty subdirectories under the top-level directories
+for BACKUP_DIR in $ROOT_PATH/backups/*; do
+  if [ -d "$BACKUP_DIR" ]; then
+    rm -rf $BACKUP_DIR/*
+  fi
+done
+
 mkdir -p $ROOT_PATH/backups/elasticsearch
+mkdir -p $ROOT_PATH/backups/elasticsearch/indices
 mkdir -p $ROOT_PATH/backups/influxdb
 mkdir -p $ROOT_PATH/backups/mongo
 mkdir -p $ROOT_PATH/backups/minio
```

Comment:
@rikukissa what about the vsexports and metabase directory?

Reply:
@euanmillar those two are created [here](https://github.com/opencrvs/opencrvs-farajaland/pull/727/files#diff-3442b11952998151cf124e279227a7b1084191d1f11dfdb691345a35d4bd8b7aR287-R292)